### PR TITLE
Dedicated data sharing col on cases page

### PR DESCRIPTION
**Pull request by northwestwitch**
Like this:

image

The icons have tooltips:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 1]
```diff
@@ -79,19 +79,20 @@
     <table class="table table-hover table-bordered">
       <thead>
         <tr>
-          <th style="width: 25%">{{ group_name|capitalize }} cases ({{cases|length}}/{{status_ncases[group_name]}})</th>
-          <th style="width: 10%">Status <a href="" id="{{group_name}}_status" onclick="updateArgs('{{group_name}}','sort=status','{{sort_option}}');" class="badge" style="color:{{'black' if sort_by=='status' else 'grey'}};background-color:white;">
+          <th class="w-25">{{ group_name|capitalize }} cases ({{cases|length}}/{{status_ncases[group_name]}})</th>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:82]: 💯

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 2]
```diff
@@ -172,33 +173,13 @@
...
-                  {% endfor %}
-                </ul>
-              </div>
-              <div class="modal-footer">
-                <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-              </div>
-            </div>
-          </div>
-        </div>
+        <span class="badge rounded-pill bg-dark" data-bs-toggle="tooltip" title="ClinVar"><em class="fas fa-database"></em></span>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:176]: 👍

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 3]
```diff
@@ -172,33 +173,13 @@
     </td>
     <td>
       {% if case.clinvar_variants %}
-        <a href="#clinvar_{{case.display_name}}" data-bs-toggle="modal">{{case.clinvar_variants|length}} {% if case.clinvar_variants|length >1 %} variants {% else %} variant {% endif %}</a>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:175]: Makes sense now that that info is square on the case page.

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 4]
```diff
@@ -294,5 +275,10 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
     // Reset search
     searchTerm.value="";
   };
+
+  var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:279]: 👍

File: scout/server/blueprints/institutes/templates/overview/cases.html (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/institutes/templates/overview/cases.html:176]: Perhaps also note in the tooltip where more info about the different submissions can be found - like on the case page, or on the institute ClinVar submissions page?

**Review by dnil — APPROVED** [review]
Nice - both a fix since the clinvar sub info is no longer on the variant page, and a feature to see MME and Beacon directly here! Consider elaborating the tooltips a little, but the modal wasn't that verbose so no essay needed.

File: scout/server/blueprints/institutes/templates/overview/cases.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/cases.html]: Good suggestion! Done!

**Pull request merged**